collect_app/src/main/java/org/odk/collect/android/utilities/ResponseMessageParser.java

```diff
@@ -0,0 +1,86 @@
+package org.odk.collect.android.utilities;
```

Comment:
Since this is a brand new class, you should be able to just run the standard code formatter from Android studio to make it nice and standard (it makes diffs really hard on existing files). Code > Reformat Code